specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-10-20/examples/WorkbookAdd.json

```diff
@@ -0,0 +1,70 @@
+{
+  "parameters": {
+    "api-version": "2020-10-20",
+    "resourceGroupName": "my-resource-group",
+    "resourceName": "deadb33f-8bee-4d3b-a059-9be8dac93960",
+    "subscriptionId": "subid",
+    "sourceId": "/subscriptions/00000000-0000-0000-0000-00000000/resourcegroups/MyGroup",
+    "workbookProperties": {
+      "id": "c0deea5e-3344-40f2-96f8-6f8e1c3b5722",
+      "location": "west us",
+      "name": "Blah Blah Blah",
+      "kind": "user",
```

Comment:
kind should be "shared" for workbooks and "user" for myworkbooks

Reply:
thanks! good to know!